src/main/java/net/pms/remote/RemoteUtil.java

```diff
@@ -175,16 +175,16 @@ public static void sendLogo(HttpExchange t) throws IOException {
 		dump(in, os);
 	}
 
-	public static boolean directmime(String mime) {
+	public static boolean directmime(String mimeType) {
 		if (
-			mime != null &&
+			mimeType != null &&
 			(
-				mime.equals(MIME_MP4) ||
-				mime.equals(MIME_WEBM) ||
-				mime.equals(MIME_OGG) ||
-				mime.equals(MIME_MP3) ||
-				mime.equals(MIME_PNG) ||
-				mime.equals(MIME_JPG)
+				mimeType.equals(MIME_MP4) ||
+				mimeType.equals(MIME_WEBM) ||
```

Comment:
MP3 ?
I already did in one of my PR, perhaps not perfectly though, but as your've more chance to be merged, it was worst a try, isn't it :wink: 

Feel free to get rest on it, or push it without the Web interface support, that can be done later, as i still never see a complain about that. Nobody will blame you for that.